modules/sdk-coin-avaxp/src/lib/iface.ts

```diff
@@ -2,6 +2,14 @@ import { Entry, TransactionExplanation as BaseTransactionExplanation, Transactio
 
 export interface TransactionExplanation extends BaseTransactionExplanation {
   type: TransactionType;
+  fromAddresses: string[];
+  rewardAddresses: string[];
+  nodeID: string | undefined;
+  totalIn: string;
+  totalStaked: string;
```

Comment:
same, are we duplicating info?

can `totalStaked` be derived from summing up values in `outputs` for staking txns?

Reply:
`totalIn` would be the staked + fee, `totalStaked` would be the sum of the outputs. This can be removed